siva/storage.go

```diff
@@ -149,6 +214,11 @@ func (s *Storage) PackfileWriter() (io.WriteCloser, error) {
 // Close finishes writes to siva file and cleans up temporary storage.
 func (s *Storage) Close() error {
 	defer s.Cleanup()
+
+	if err := s.PackRefs(); err != nil {
+		return err
+	}
+
 	return s.Sync()
```

Comment:
It's better to always execute sync otherwise the siva file will be unreadable if `PackRefs` fails.